src/Dropdown.js

```diff
@@ -71,10 +71,10 @@ const propTypes = {
 
   /**
    * A callback fired when the Dropdown wishes to change visibility. Called with the requested
-   * `open` value.
+   * `open` value and the type of event that fired it: `'click'`,`'keydown'`,`'close'`, or `'select'`.
```

Comment:
It was already being passed from handleKeyDown

![image](https://cloud.githubusercontent.com/assets/12389411/21755738/1d536396-d5cd-11e6-9881-debbf6f276ad.png)